package.json

```diff
@@ -50,7 +50,7 @@
     "devDependencies": {
         "cross-env": "^7.0.3",
         "docdash": "^1.2.0",
-        "enketo-transformer": "1.42.0",
+        "enketo-transformer": "https://github.com/eyelidlessness/enketo-transformer.git#1617062",
```

Comment:
As far as I can tell from doing a quick `blame`, @MartijnR has always done a Transformer release before merging something to dependent repos. Now that we're looking at a larger and less coordinated set of devs who don't all have permissions to do releases, I think we should pick a process and document it. E.g. Keep PRs like this one in draft state until all Transformer PRs have been reviewed and merged, request a Transformer release from @MartijnR, then update the Transformer version and mark PR as ready.

Reply:
This makes sense! I've converted this and #779 to draft PRs. <small>(Could they make that link-button any more invisible?)</small>